app/__init__.py

```diff
@@ -274,6 +276,12 @@ def update_sent_state(sender=None, headers=None, **kwargs):
 scheduler.add_job(expire_pending_tickets, 'cron', minute=45)
 scheduler.add_job(send_monthly_event_invoice, 'cron', day=1, month='1-12')
 scheduler.add_job(event_invoices_mark_due, 'cron', hour=5)
+with current_app.app_context():
+    try:
+        order_expiry_time = get_settings()['order_expiry_time']
+    except ProgrammingError:
+        order_expiry_time = 15
```

Comment:
TOo error-prone, hardcode 10 minutes

Reply:
I used the default value that was used in model.

Reply:
This can mean that a ticket will not get expired for at most 30 minutes. We don't want that. Changing to 10 minutes, it'll be deleted after at max 25 minutes. Changing it to 5 means it will be deleted after at max 20 minutes. So change it to 5